packages/react-native-bpk-component-calendar/src/ios/CalendarBridge/RCTBPKCalendar.m

```diff
@@ -68,28 +68,31 @@ - (void)setMinDate:(NSDate *)minDate {
     NSDate *localDate = [RCTBPKCalendarDateUtils convertDateToLocal:minDate
                                                       localCalendar:self.gregorian
                                                         utcCalendar:self.utcCalendar];
-    [super setMinDate:localDate];
+    BPKSimpleDate *simpleDate = [super simpleDateFromDate:localDate];
```

Comment:
Should we consider having both a `simpleDateFromLocalDate` and `simpleDataFromUTCDate`?

Reply:
If so we should include this in https://github.com/Skyscanner/backpack-ios/pull/203

Reply:
Possibly, but the "LocalDate" naming might cause confusion if used on the public interface. In the Skyscanner app we are calling the timezone independent dates LocalDate and its not just a custom name, we are following the naming of java.Time.LocalDate:
https://docs.oracle.com/javase/8/docs/api/java/time/LocalDate.html
"A date without a time-zone in the ISO-8601 calendar system"

So technically this would be something like simpleDateFromDateInLocalTimeZone, not sure there is a more compact way to name it. :)
What do you think, would you still change the current simpleDateFromDate to these 2?

Reply:
Nah let's keep it as is and use the conversion from UTC to local time zone here before calling `simpleDateFromDate:`